### PR TITLE
feat(replay): Render an unstyled Replay Preview for Replay Issues

### PR DESCRIPTION
**Pull request by ryan953**
The purpose of this PR is to implement some feature checks so we can work inside them. 

When `organizations:issue-details-replay-event` is deployed & enabled you'll see an unstyled `<ReplayPreview>`:

SCR-20230607-oxci

With these initial checks in place i'm going to follow up with some refactors to the ReplayPreview, and style the thing for this position in the page.

Can test by opening the new issue type.
Example: https://sentry.dev.getsentry.net:7999/issues/4229046598/events/bdbfa6060685497187ae716e5a3e196d/?project=11276&referrer=previous-event

Related to:
* https://github.com/getsentry/team-replay/issues/91
* https://github.com/getsentry/sentry/issues/48259

### CODE REVIEW

File: static/app/utils/issueTypeConfig/replayConfig.tsx
Comment (ryan953) [inline static/app/utils/issueTypeConfig/replayConfig.tsx:1]: I'm leaving most of these enabled, even though they might not work yet, so we can see them and fix them.

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
We are pulling back on dead click as a replay issue, so not sure how we want to go forward with this.

**Comment by ryan953** [comment]
> We are pulling back on dead click as a replay issue, so not sure how we want to go forward with this.

kk. I'll let it marinate for a bit.

**Comment by cmanallen** [comment]
Rage clicks might make the cut.  But who knows.  We'll see after we gather more data.

**Pull request closed without merging**